app/src/main/java/org/oppia/app/profile/ProfileActivityPresenter.kt

```diff
@@ -26,7 +26,7 @@ class ProfileActivityPresenter @Inject constructor(
       allowDownloadAccess = true,
       colorRgb = -10710042,
       isAdmin = true,
-      storyTextSize = StoryTextSize.SMALL_TEXT_SIZE,
+      storyTextSize = StoryTextSize.MEDIUM_TEXT_SIZE,
```

Comment:
Is the default for regular new profiles correct? Are there tests guarding this?

Reply:
@BenHenning I have updated the description. PTAL.